data/actionsmap/yunohost.yml

```diff
@@ -925,6 +925,54 @@ monitor:
 
 
 #############################
+#         Settings          #
+#############################
+settings:
+    category_help: Manage YunoHost global settings
+    actions:
+
+        ### settings_list()
+        list:
+            action_help: list all entries of the settings
+            api: GET /settings
+
+        ### settings_get()
+        get:
+            action_help: get an entry value in the settings
+            api: GET /settings/<key>
+            arguments:
+                key:
+                    help: Settings key
+
+        ### settings_set()
+        set:
```

Comment:
After discussing this on the chat with @Psycojoker, we thought about using "decorators" to be able to track which (python) functions should be triggered when changing the value of a setting. This should be relatively easy to implement and add later. Same goes for bash/'external' hooks if we want to go in that direction.